src/widget/systemtrayicon.cpp

```diff
@@ -99,6 +99,7 @@ SystemTrayIcon::SystemTrayIcon()
 
 SystemTrayIcon::~SystemTrayIcon()
 {
+    delete qtIcon;
     qDebug() << "Deleting SystemTrayIcon";
 }
```

Comment:
I'm not sure but maybe they should make the destruction of the other pointer (with validation)
```
    SystrayBackendType backendType;
    QSystemTrayIcon* qtIcon;

#ifdef ENABLE_SYSTRAY_UNITY_BACKEND
    AppIndicator *unityIndicator;
    GtkWidget *unityMenu;
#endif
#ifdef ENABLE_SYSTRAY_STATUSNOTIFIER_BACKEND
    StatusNotifier* statusNotifier;
    GtkWidget* snMenu;
#endif
#ifdef ENABLE_SYSTRAY_GTK_BACKEND
    GtkStatusIcon* gtkIcon;
    GtkWidget* gtkMenu;
#endif
```